includes/media.php

```diff
@@ -88,12 +88,14 @@ function pods_image( $image, $size = 'thumbnail', $default = 0, $attributes = ''
 	$default = pods_image_id_from_field( $default );
 
 	if ( 0 < $id ) {
-		if ( $force ) {
+		if ( $force && 'full' !== $size ) {
 			$full = wp_get_attachment_image_src( $id, 'full' );
-			$src  = wp_get_attachment_image_src( $id, $size );
+			if ( ! empty( $full[0] ) ) {
+				$src  = wp_get_attachment_image_src( $id, $size );
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 2 spaces

<sub>hash: e33a73278c2abcca849d57ce5bbef116</sub>